### PR TITLE
[TH] add read and search queues

### PR DESCRIPTION
**Pull request by tank157**
Not strictly called for yet, but nice to have
to demo the delete fns. So there isn't a Pivotal
Card for this.

Assigning to @cap10morgan

### CODE REVIEW

File: src/scheduled_message_works/handlers.clj [hunk 1]
```diff
@@ -89,3 +94,19 @@
...
     (delete-message-by-identifiers (:identifiers identifying-data))
     (delete-message-by-uuid (:id identifying-data))))
+
+(defn read-message
+  [message-data]
+  (s/validate uuid-schema message-data)
+  (if-let [message (message/find-by-id (db/db) (:id message-data))]
+    {:status :ok
+     :message (message/entity->message message)}
+    (throw (IllegalArgumentException. "Not found"))))
```
Comment (cap10morgan) [inline src/scheduled_message_works/handlers.clj]: Should we include the id that it failed to find in the error message?

Comment (tank157) [inline src/scheduled_message_works/handlers.clj]: We haven't been doing so anywhere else we're doing the same thing yet. It's a direct response type of call, so the caller should have the id already. But I could be convinced we need to adjust our informal standard around this if you think it's still necessary.

Comment (cap10morgan) [inline src/scheduled_message_works/handlers.clj]: I'm just thinking about when we're tailing logs in production later. Getting the id there is really helpful.

Comment (cap10morgan) [inline src/scheduled_message_works/handlers.clj]: But I see that right now handle-errors isn't logging these. So fine, we can fix that later at a different level (something like handle-errors feels destined for a library of some sort).

**Comment by cap10morgan** [comment]
LGTM

**Comment by cap10morgan** [comment]
Oops, sorry, one more question: Should we add anything to the README about these changes?

**Comment by tank157** [comment]
Ok, I've updated the README and added the id to the Not Found case. One more quick look @cap10morgan ?

**Comment by cap10morgan** [comment]
LGTM

**Pull request merged**